client/app/bundles/course/courses/pages/CoursesIndex/index.tsx

```diff
@@ -50,7 +51,14 @@ const translations = defineMessages({
 
 const CoursesIndex: FC = () => {
   const { t } = useTranslation();
-  const [isNewCourseDialogOpen, setIsNewCourseDialogOpen] = useState(false);
+
```

Comment:
is this intentional?^

Reply:
To not centre the course logos? Yes. Since the description is left-aligned, I made it so that the logo and course name are also left-aligned. Then everything is left aligned.

It also doesn't look too weird when there isn't any descriptions.

Reply:
ok!